src/validators/satelliteIdValidator.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response, NextFunction } from 'express'
+import { assert, pattern, string } from 'superstruct'
+
+export const validateSatelliteIdRoute = (req: Request, res: Response, next: NextFunction):
+    //ReturnType<NextFunction> | ReturnType<typeof response.sendStatus> => {
+    void => {
+    const idCheck = pattern(string(), /[0-9]+/)
```

Comment:
"0123" will pass through this validation. Consider changing it to: ```/^[1-9][0-9]+$/``` which ensures that the first digit is not zero.